schemas/iso19139/src/main/plugin/iso19139/layout/layout-custom-fields.xsl

```diff
@@ -42,15 +42,31 @@
 
   <!-- Readonly elements -->
   <xsl:template mode="mode-iso19139" priority="2100" match="gmd:fileIdentifier|gmd:dateStamp">
+    <xsl:param name="schema" select="$schema" required="no"/>
+    <xsl:param name="labels" select="$labels" required="no"/>
+    <xsl:param name="overrideLabel" select="''" required="no"/>
 
     <xsl:variable name="xpath" select="gn-fn-metadata:getXPath(.)"/>
     <xsl:variable name="isoType" select="if (../@gco:isoType) then ../@gco:isoType else ''"/>
-    <xsl:variable name="labelConfig"
+    <xsl:variable name="fieldLabelConfig"
                   select="gn-fn-metadata:getLabel($schema, name(), $labels, name(..), $isoType, $xpath)"/>
 
+    <xsl:variable name="labelConfig">
```

Comment:
I think this kind of code is used in several places. An option can be to update `gn-fn-metadata:getLabel` to send an optional parameter with the override label and add this login as part of  `gn-fn-metadata:getLabel` ?

Reply:
> An option can be to update `gn-fn-metadata:getLabel` to send an optional parameter with the override label and add this login as part of `gn-fn-metadata:getLabel` ?


I've no time to do that now but sure that would be an improvement.

Reply:
Ok, we can do that later as an enhancement